src/CloudModelGenerator/CommandLineOptionsProvider.cs

```diff
@@ -14,16 +15,22 @@ public CommandLineOptionsProvider(IEnumerable<Argument> appOptions)
         {
             foreach (var commandOption in appOptions)
             {
-                if (commandOption.Value == null)
+                if (commandOption.Value != null)
                 {
-                    continue;
-                }
+                    string value = commandOption.Value.ToString();
 
-                string value = commandOption.Value.ToString();
+                    if (!string.IsNullOrEmpty(value))
+                    {
+                        var paramName = commandOption.Names.Last();
 
-                if (!string.IsNullOrEmpty(value))
-                {
-                    Data.Add(commandOption.Names.Last(), value);
+                        // Backward compatibility
```

Comment:
Do we really want this backward compatibility?

If yes - I would probably link the issue and cover the functionality by the test...
```suggestion
                        /// Backward compatibility <see href="https://github.com/Kentico/cloud-generators-net/issues/69"/>
```